dds/idl/v8_generator.cpp

```diff
@@ -343,7 +343,7 @@ namespace {
           "  {\n"
           "    v8::Local<v8::Value> lv = " << src << "->Get(" << prop << ");\n"
           "    v8::Local<v8::Object> lo = Nan::To<v8::Object>(lv).ToLocalChecked();\n"
-          "    copyFromV8(lo, " << propName << ");\n"
+          "    copyFromV8(lo, " << (fun_assign ? propName + "()" : propName) << ");\n"
```

Comment:
`propName` is on both sides of the conditional, can it be moved out? (also on line 355)

Reply:
I was trying to avoid the empty string, but I can move the final ) in, in order to remove the propName redundancy.